src/components/Icon/MucIcon.vue

```diff
@@ -1,15 +1,19 @@
 <template>
   <svg
     aria-hidden="true"
-    class="m-button__icon"
+    :class="[additionalClasses, iconClass]"
```

Comment:
Can we not simply use `class` as a property? Having two properties do the same thing seems kind of complicated. A user can set a string to `class` which is seperated by spaces - and thus I think we only need one property.

Reply:
I don't think it needs to be an array. What I imagine would be:

```html
<muc-icon class="class-a class-b class-c" />
```

Reply:
Should be resolved - used `Fallthrough Attributes` now with computed fallback value